src/main/java/core/basesyntax/FiguresSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FiguresSupplier {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        return new Circle(Color.WHITE.name(), 10);
+    }
+
+    public Figure getRandomFigure() {
+        int randomSize = random.nextInt(10);
```

Comment:
The random size is generated with 'random.nextInt(10)', which will produce sizes from 0 to 9. This means some figures might have a size of 0, which may not be valid for all figures. Consider setting a minimum size greater than 0. Avoid using magic numbers like `10` in this example, check this link: https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#all-magic-numbers-in-your-code-should-be-constants